lldpad.c

```diff
@@ -262,7 +262,7 @@ int main(int argc, char *argv[])
 			pid_file = 0;
 			break;
 		case 'P':
```

Comment:
@imanti please change this to 'R'

Reply:
Done!

Reply:
@imanti did you push the change? I don't see here.

Reply:
I did. I don't know why if you go directly to the last commit the changes are not updated. If you check the PR files changed tab you can see the change.